extract-webidl.js

```diff
@@ -0,0 +1,56 @@
+var jsdom = require('jsdom');
+
+function extract(url, cb) {
+    jsdom.env(url, [],
+              function(err, window) {
+                  if (err) return cb(err);
+                  var generator = window.document.querySelector("meta[name='generator']");
+                  if (generator && generator.content.match(/bikeshed/i)) {
+                      return extractBikeshedIdl(window.document, cb);
+                  }
+                  if (window.document.getElementById('respecDocument')) {
+                      return extractRespecIdl(window.document, cb);
+                  }
```

Comment:
So, that only works for generated version of ReSpec documents.

See code in https://github.com/tidoust/reffy/commit/4873d9d2c3289b3153db358ece7bd76051e1d775 for a way to make JSDom run ReSpec and attach this code to after ReSpec is done generating the document.

Feel free to merge as-is, I'll make the update afterwards.